File: backend/src/test/kotlin/fr/amory/libris/fixture/SourceEditions.kt
package fr.amory.libris.fixture

import fr.amory.libris.domain.Kind.BOOK
import fr.amory.libris.domain.lookup.SourceEdition

val A_SOURCE_EDITION = SourceEdition(
    isbn = isbnOf("9782723488525"),
    kind = BOOK,
    title = "Un ouvrage",
    subtitle = null,
    authors = emptyList(),
    series = null,
    collection = null,
    publisher = null,
    publicationYear = null,
    language = null,
    pageCount = null,
    summary = null,
    coverUrl = null,
)

File: frontend/src/fixture/SourceEditions.ts
import type { SourceEdition } from "../domain/SourceEdition";

export const onePiece1: SourceEdition = {
  isbn13: "9782723488525",
  kind: "MANGA",
  title: "Romance dawn",
  subtitle: "à l'aube d'une grande aventure",
  authors: [
    { name: "Eiichirō Oda", role: "WRITER" },
    { name: "Eiichirō Oda", role: "ARTIST" },
  ],
  series: { name: "One piece", volumeNumber: 1 },
  collection: "Shonen manga",
  publisher: "Glénat",
  publicationYear: 2013,
  language: "fr",
  pageCount: 203,
  summary: null,
  coverUrl: "https://covers.openlibrary.org/b/isbn/9782723488525-L.jpg",
};

export const onePiece2: SourceEdition = {
  isbn13: "9782723489898",
  kind: "MANGA",
  title: "Aux prises avec Baggy et ses hommes",
  subtitle: null,
  authors: [
    { name: "Eiichirō Oda", role: "WRITER" },
    { name: "Eiichirō Oda", role: "ARTIST" },
  ],
  series: { name: "One piece", volumeNumber: 2 },
  collection: "Shonen manga",
  publisher: "Glénat",
  publicationYear: 2013,
  language: "fr",
  pageCount: 208,
  summary: null,
  coverUrl: "https://covers.openlibrary.org/b/isbn/9782723489898-L.jpg",
};
